es-vue-base/src/lib-components/es-ds-vue-sample.vue

```diff
@@ -1,77 +1,80 @@
 <script>
```

Comment:
I'm assuming this is just prettier updates. FWIW I forgot to delete this file. Hopefully there wasn't any manual work updating it.

Reply:
Yep, it's gone in another PR.